src/nunit-gui/Views/TestTreeView.cs

```diff
@@ -62,6 +67,8 @@ public TestTreeView()
 
             RunContextCommand = new MenuElement(this.runMenuItem);
             RunCheckedCommand = new MenuElement(this.runCheckedMenuItem);
+            DebugContextCommand = new MenuElement(this.debugMenuItem);
```

Comment:
I know you are following suite with using `this.` and there appears to be mixed in here. Can you remove all `this.`s in this file only please.